CONTRIBUTING.md

```diff
@@ -51,6 +51,79 @@ For source code integrity , Hyperledger Bevel GitHub pull requests are accepted
+   # Checkout the main branch - you want your new branch to come from main
+   git checkout main
+   # Create a new branch named `<newfeature>` (give simple informative name)
+   git branch <newfeature>
+   ```
+4. Checkout your branch and add/modify files.
+   ```
+   git checkout <newfeature>
+   git rebase main
+   # Happy coding !
+   ```
+5. Commit changes to your branch.
+   ```
+   # Commit and push your changes to your fork
+   git add -A
+   git commit -s -m "<type>[optional scope]: <description>"
+   git push origin <newfeature>
+   ```
+6. Once you've committed and pushed all of your changes to GitHub, go to the page for your fork on GitHub, select your development branch, and click the pull request button.
+
+7. Repeat step 3 to 6 when you need to prepare posting new pull request.
+
+NOTE: Once you submitted pull request to Bevel repository, step 6 is not necessary when you made further changes with `git commit --amend` since your amends will be sent automatically.
+
+NOTE: You can refer original tutorial ['GitHub Standard Fork & Pull Request Workflow'](https://gist.github.com/Chaser324/ce0505fbed06b947d962)
+
+### Directory structure
+
+Whenever you begin to use your codes on Hyperledger Bevel, you should follow the directory strecture on Hyperledger Bevel.
+The current directory structure is described as the following:
+
+
+> - automation/ : This folder contains the Input configuration file and Jenkinsfile for r3-corda, hyperledger fabric which helps in automation of corda/fabric network.
+>   - hyperledger-fabric/ : ontains network.yaml file and fabric Jenkinsfile
+>   - r3-corda/ : Contains network.yaml file and Jenkinsfile for doorman,networkmap and entire network deployment file.
+>   - initcontainer.Jenkinsfile : This file is used to initialize a container with basic configuration. This will be used on both hyperledger and r3-corda platforms.
+> - docs/
+>    - source/
+>         - index.rst : This is the main document. Main function of this document is to serve as a welcome page, and to contain the root of the “table of contents tree” (or toctree).This is one of the main things that Sphinx adds to reStructuredText, a way to connect multiple files to a single hierarchy of documents
+>         - conf.py : The configuration directory must contain a file named conf.py. This file (containing Python code) is called the “build configuration file” and contains (almost) all configuration needed to customize Sphinx input and output behavior.
+>         - *.md : Create all the markdown file which are referenced in the document tree with the appropriate content.
+>    - Makefile
+>    - pip-requirements.txt
+>    - README.md     
+> - platforms/
+>     - hyperledger-besu/
+>     - hyperledger-fabric/ 
+>     - hyperledger-indy/ 
+>     - quorum/ 
+>     - r3-corda-ent/
+>     - r3-corda/ 
+>     - shared/ : The shared folder contains various sub folders which in turn each have certain types of files that will be used as prerequisites
```

Comment:
The description of this is wrong.
Please change it to something like
`This folder contains the master playbook (site.yaml), along with some pre-requisite roles, playbooks and other shared roles which are commonly used by all the platforms`